app/actions/definitions/trash.tsx

```diff
@@ -0,0 +1,28 @@
+import { TrashIcon } from "outline-icons";
+import React from "react";
+import DeleteDocumentsInTrash from "~/scenes/Trash/components/DeleteDocumentsInTrash";
+import { createAction } from "~/actions";
+import { TrashSection } from "~/actions/sections";
+import { trashPath } from "~/utils/routeHelpers";
+
+export const permanentlyDeleteDocumentsInTrash = createAction({
```

Comment:
Sorry, I didn't notice the first round of review – this belongs in `actions/definitions/documents` just fine as we're acting on documents here.